src/clj/rems/db/test_data.clj

```diff
@@ -128,18 +129,14 @@
 
 (defn create-catalogue-item! [{:keys [title resource-id form-id workflow-id]
                                :as command}]
-  (let [result (catalogue/create-catalogue-item! {:resid (or resource-id (create-resource! {}))
-                                                  :form (or form-id (create-form! {}))
-                                                  :wfid (or workflow-id (create-dynamic-workflow! {}))})
-        _ (assert (:success result) {:command command :result result})
-        id (:id result)]
-    (when (map? title)
-      (doseq [[lang text] title]
-        (let [result (catalogue/create-catalogue-item-localization! {:id id
-                                                                     :langcode (name lang)
-                                                                     :title text})]
-          (assert (:success result) {:command [id lang text] :result result}))))
-    id))
+  (let [localizations (map-vals (fn [title] {:title title}) title)
+        result (catalogue/create-catalogue-item!
+                {:resid (or resource-id (create-resource! {}))
+                 :form (or form-id (create-form! {}))
+                 :wfid (or workflow-id (create-dynamic-workflow! {}))
+                 :localizations (or localizations {})})
+        _ (assert (:success result) {:command command :result result})]
```

Comment:
[nit] no need for the `_` trick here, the assert can just be in the body of the let

Reply:
Done.